### DISCUSSION TITLE
Feature Request: `Automerge::clone_without_history(&other_document:<Automerge>)`

### DISCUSSION THREAD

**Discussion by issackelly** [automerge/automerge#1527] [General]
I would like to snapshot the history at a given point. I haven't dug through all the edges yet but I think that doing this reliably with something like @gterzian 's work on https://medium.com/@polyglot_factotum/distributing-lamports-bakery-with-automerge-and-a-touch-of-tla-679b2705b7cc would let me implement tomb-stoning in a reliable way in a distributed system and could be a convenient backup, branching or snapshot mechanism.

**Comment by alexjg** [comment]
This seems like a reasonable feature request. I think we could implement it on `ReadDoc` so that you can do it with either `Automerge` or `AutoCommit`.

**Comment by FintanH** [comment]
I've moved this here so that the potential design of the feature can be discussed before having an implementation.
